apps/frontend/src/libs/components/button/styles.module.css

```diff
@@ -24,7 +24,7 @@
 	background-color: var(--brand);
 }
 
-.secondary {
+.secondary-button {
```

Comment:
why do we need this -button suffix?

Reply:
I think its button variant style.

![Screenshot (371)](https://github.com/user-attachments/assets/b3a71b44-371a-465e-a863-e976f3e1048a)